### PR TITLE
ref(issue-alert): add test notifications and preview

### PR DESCRIPTION
**Pull request by Zandew**
Adds some documentation for notification tests and rule previews.

### CODE REVIEW

File: src/docs/product/alerts/create-alerts/issue-alert-config.mdx [hunk 1]
```diff
@@ -110,6 +114,10 @@ The available intervals are:
 - Hours: 3, 12, 24
 - Days: 7, 30
 
+## Alert Preview
+
+Displays what issues would have triggered a rule with your current conditions and when they would have last triggered it. Previews cannot be generated for certain rule sets and previews containing issue frequency conditions will be approximated.
```
Comment (imatwawana) [inline src/docs/product/alerts/create-alerts/issue-alert-config.mdx:119]: We should explain, at least at a high level, which alert rules won't generate preview.
```suggestion
The preview displays a list of issues that would have triggered the alert you've configured and when they would have last triggered it. Previews can't be generated for certain alert rules and previews containing issue frequency conditions are approximated.
```

**Review by imatwawana — COMMENTED** [review]
I've left a recommendation for further clarification on the alert preview as well as some wording edits.

File: src/docs/product/alerts/create-alerts/issue-alert-config.mdx [hunk 2]
```diff
@@ -116,7 +116,14 @@ The available intervals are:
 
 ## Alert Preview
 
-The preview displays a list of issues that would have triggered the alert you've configured and when they would have last triggered it. Previews can't be generated for certain alert rules and previews containing issue frequency conditions are approximated. Support for certain low priority conditions have not yet been added, as well as rules that use both frequency conditions and event filters.
+The preview displays a list of issues that would have triggered the alert you've configured and when they would have last triggered it.
```
Comment (imatwawana) [inline src/docs/product/alerts/create-alerts/issue-alert-config.mdx:119]: Realized I was still finding this sentence a bit hard to parse
```suggestion
The preview displays a list of issues that would have triggered the alert and the last time they would have triggered it.
```

**Review by imatwawana — COMMENTED** [review]
Okay this is great! One more question and once that's clear, this is good to go.

File: src/docs/product/alerts/create-alerts/issue-alert-config.mdx [hunk 3]
```diff
@@ -116,7 +116,14 @@ The available intervals are:
...
+
+Rules with any of following are not previewable:
+- The issue is assigned to {no one/a team/a member}.
+- The event is from the latest release.
+- An issue affects more than [{X} percent of sessions in {time}](#percent-based-alerts)
+- No ["When" conditions](#when-conditions-triggers)
```
[the remark this answers is not in the document]
Comment (Zandew) [reply src/docs/product/alerts/create-alerts/issue-alert-config.mdx:125]: sorry for the double negative lol, I mean the former

File: src/docs/product/alerts/create-alerts/issue-alert-config.mdx [hunk 4]
```diff
@@ -110,6 +114,17 @@ The available intervals are:
...
+## Alert Preview
+
+The preview displays a list of issues that would have triggered the alert and the last time they would have triggered it.
+
+Rules with any of following can't be previewed:
+- The issue is assigned to {no one/a team/a member}.
+- The event is from the latest release.
+- An issue affects more than [{X} percent of sessions in {time}](#percent-based-alerts)
+- No ["When" conditions](#when-conditions-triggers)
+- [Issue frequency](#change-alerts) conditions and [event filters](#if-conditions-filters) (filters that start with "The event")
```
Comment (imatwawana) [inline src/docs/product/alerts/create-alerts/issue-alert-config.mdx:126]:
```suggestion
- Both [issue frequency](#change-alerts) conditions and [event filters](#if-conditions-filters) (filters that start with "The event")
```

File: src/docs/product/alerts/create-alerts/issue-alert-config.mdx (hunk 3 above)
[the remark this answers is not in the document]
Comment (imatwawana) [reply src/docs/product/alerts/create-alerts/issue-alert-config.mdx:125]: Okay I think we can just leave it as is!

**Review by imatwawana — APPROVED** [review]
Left one last tiny wording nitpick, but otherwise this LGTM! Thanks for all the work to refine this.
